airbyte-ci/connectors/connector_ops/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "connector_ops"
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
@aaronsteers please add a changelog entry in https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md#L403

We unfortunately forgot to update it recently :(

Reply:
also my change! Ill break this out into its own branch and get it in ahead of duckdb

Reply:
Hey @aaronsteers!

Sorry for not elaborating here.

Based on the link I had infered that @alafanechere was infact meaning to comment on the pipelines pyproject.toml
https://github.com/airbytehq/airbyte/blob/destination-duckdb/feat-motherduck-support/airbyte-ci/connectors/pipelines/pyproject.toml#L7

At the time I did not have a changelog entry but have since updated the branch to have a changelog entry.

So we should be able to consider this resolved. :) 

Also while were elaborating! I want to bring up why I did not break these changes out into a separate branch. Essentially to test that our pyproject.toml interpretation and install logic was working as expected the quickest way was to extend [this test](https://github.com/airbytehq/airbyte/blob/destination-duckdb/feat-motherduck-support/airbyte-ci/connectors/pipelines/tests/test_actions/test_environments.py#L19) to include a connector based on pyproject.toml (destination-duckdb).

So there was a slight chicken and egg problem (pun intended) that had me leaving the code in this PR

Reply:
Makes sense, @bnchrch! And thanks again!